### PR TITLE
Create crud functionality for games

### PR DESCRIPTION
**Pull request by Jorbreezy**

### CODE REVIEW

File: routes/createGames.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var express = require('express');
+var router = express.Router();
+
+const {
+ postGame
+} = require('../controllers/games.controller');
+
+router.get('/compose/new', (req, res) => {
```
Comment (joelcolucci) [inline routes/createGames.js]: Nice! What's your thinking on going with this route naming convention?

File: app.js [hunk 2]
```diff
@@ -21,6 +22,7 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', indexRouter, gamesRouter, studioRouter);
+app.use('/games', createRouter)
```
Comment (joelcolucci) [inline app.js]: What are your thoughts on having the games routing split between two route files?

File: routes/createGames.js (hunk 1 above)
Comment (Jorbreezy) [reply routes/createGames.js]: I looked around at other big websites naming for submitting a post or email and decided to go with it.

File: app.js (hunk 2 above)
Comment (Jorbreezy) [reply app.js]: I don't necessarily have any thoughts but I could've gone farther and separated them by mutations and read. That way its sorta more organized.

File: routes/createGames.js (hunk 1 above)
Comment (joelcolucci) [reply routes/createGames.js]: Nice work checking out other sites. I think for our use case this naming convention doesn't create the cohesion we want it to.

The general standard is:
```
/games/:id
/games
/games/new
/games/:id/edit
/games/:id/delete

```

File: app.js (hunk 2 above)
Comment (joelcolucci) [reply app.js]: I would go ahead and separate them entirely. We want our developers to be able to go to one place to work on the games resource.

Comment (Jorbreezy) [reply app.js]: Wait separate or combine them?

**Review by joelcolucci — APPROVED** [review]
Nice work!

**Pull request merged**